### ISSUE TITLE
Shortcut for activating orca isn't set by default

### ISSUE THREAD

**Issue by pavelvlcek** [linuxmint/Cinnamon#5592]
Dear developers,
blind users can't activate Orca screen reader without sighted person now, because the shortcut for enabling Orca screen reader isn't set by default. It will be good to set it for example to alt plus super plus s key.

**Comment by xenopeek** [comment]
I don't think this is an issue in Cinnamon itself but an issue with the Linux Mint distribution with the Cinnamon desktop environment installed.

**Comment by Vahan86** [comment]
The issue still exists in Cinnamon 4.4.

**Comment by Fantu** [comment]
https://github.com/linuxmint/cinnamon-desktop/pull/198 close this
